spock-core/src/main/java/org/spockframework/compiler/WhereBlockRewriter.java

```diff
@@ -16,21 +16,27 @@
 
 package org.spockframework.compiler;
 
-import java.util.*;
```

Comment:
The imports are done by Idea automatically.

The end of line trimming is an `.editorconfig` setting, provided with the project: `trim_trailing_whitespace = true`

Would it be possible to configure these setting the way you would like to receive them, and reformat the whole codebase with it (and commit the config also).

It shouldn't take more than an hour, and would save all these reviews about formatting.